onyo/main.py

```diff
@@ -376,9 +335,16 @@ def setup_parser():
         help='Open new assets to edit them before creation'
     )
     cmd_new.add_argument(
-        'asset',
+        '-k', '--keys',
+        required=False,
+        action=StoreKeyValuePairs,
+        metavar="KEYS",
+        nargs='*',
```

Comment:
`--keys` should require at least one arg, thus `+` should be used.

https://docs.python.org/3/library/argparse.html#nargs

Reply:
Oh dang, you are right.